classes/04class/exercises/c04-iac02/ANSWER.md

```diff
@@ -0,0 +1,16 @@
+# C04-IAC02
+
+## Terraform code 
+- [_backend.tf](_backend.tf)
+- [_provider.tf](_provider.tf)
+- [vpc.tf](vpc.tf)
+- [output.tf](output.tf)
+- [main.tfvars](main.tfvars)
+
+## Command Execution Output
+- [output.txt](output.txt)
+
+<!-- Don't change anything below this point-->
+<!-- Before commiting, remove both commented lines--> 
+***
+Answer for exercise [c04-iac02](commit_sha to be added)
```

Comment:
same here, add some instructions on how to get the commit_sha and why we need this

Reply:
or maybe even adding this kind of information to  the main README and just use as a reference here on the exercises.

Reply:
Once we have the class and the exercises merged, Ill update the ANSWER files with the correct link so the students don't need to worry about this.
The ANSWER.md template will be filled with the right exercise commit.